tests/generate_fixtures/generate_pyani_anim_matrices.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
I would use as ``#!/bin/bash`` per http://redsymbol.net/articles/unofficial-bash-strict-mode/ but https://stackoverflow.com/questions/10376206/what-is-the-preferred-bash-shebang has some good discussion as there are pros-and-cons. This works on macOS and most if not all Linux.